### PR TITLE
 XML submittedName allowed in Uniprot. Fixes #1028

### LINKED ISSUE #1028: UniProt-XML parsing fullName as description
## Error Description
UniProt-XML **descriptions** are not properly parsed by **SeqIO**.
Parsing the entries below does not provide a description for the protein **D4VF52**.

## How to Reproduce
wget [http://www.uniprot.org/uniprot/R5HY77.xml](url)
wget [http://www.uniprot.org/uniprot/D4VF52.xml](url)

Both proteins have a _fullName_ annotation, in addition to the _recommendedName_/_submittedName_ tags:
**R5HY77.xml**: `<fullName evidence="1">Elongation factor Ts</fullName>`
**D4VF52.xml**: `<fullName evidence="4">GTP pyrophosphokinase, (P)ppGpp synthetase II / Guanosine-3',5'-bis(Diphosphate) 3'-pyrophosphohydrolase</fullName>`

```python
for entry in SeqIO.parse(open("R5HY77.xml"), "uniprot-xml"):
  print entry.id, entry.description
```
returns: `R5HY77 Elongation factor Ts`
&nbsp;

```python
for entry in SeqIO.parse(open("D4VF52.xml"), "uniprot-xml"):
  print entry.id, entry.description
```
returns: `D4VF52 <unknown description>`
&nbsp;

## Biopython Version
1.68

### PR DESCRIPTION
**Pull request by OscarMaestre**
XML submittedName wasn't parsed correctly. Now it works, which fixes bug #1028 . It also adds the test to suite. Test is also conformant to pep8 rules. I also needed to add a SwissProt file to the suite of tests (is it allowed?)

### CODE REVIEW

**Comment by peterjc** [comment]
This looks like a sensible fix, and yes, including another example XML file from UniProt is sensible here.

Are you happy to dual licence your contributions under both the _Biopython License Agreement_ **and** the _BSD 3-Clause License_? See #898.

Also, are you happy to be thanked by name in the ``NEWS`` and ``CONTRIB`` files?

**Comment by OscarMaestre** [comment]
Hello!

Yes, I agree to dual licensing all my contributions. Feel also free to 
use my name wherever you need/want.

:)

**Comment by peterjc** [comment]
Great - I'm currently on leave so won't do this immediately, but this looks ready to accept - thank you!

I'd add your name to the ``NEWS`` file (alphabetical by first name, noting you are a new contributor to the upcoming release) and ``CONTRIB`` file (alphabetical by surname, with link to your GitHub profile) - or you can do that on this pull request?

**Comment by OscarMaestre** [comment]
I added my name to both files (NEWS and CONTRIB), don't worry. And thank you :)

File: CONTRIB [hunk 1]
```diff
@@ -169,3 +169,4 @@ Christian Zmasek <https://github.com/cmzmasek>
 Xiaoyu Zhuo <https://github.com/xzhuo>
 Harry Zuzan <iliketobicycle at domain yahoo.ca>
 Bertrand Néron < https://github.com/bneron>
+Oscar G. Garcia <https://github.com/oscarmaestre>
```
Comment (peterjc) [inline CONTRIB]: This is meant to be alphabetical by surname - it looks like Bertrand's name is in the wrong place though.

Comment (OscarMaestre) [inline CONTRIB]: Yes, you're right. I fixed it

**Comment by peterjc** [comment]
I also realised there was a duplicate in the CONTRIB file: https://github.com/biopython/biopython/commit/da6440d8b0eaea1f9ed63350100ed61786dcc4ba

**Comment by peterjc** [comment]
Merged, thank you :)

**Pull request merged**